sdk/core/Azure.Core.Experimental/src/Azure.Core.Experimental.csproj

```diff
@@ -1,4 +1,4 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
nit: are there changes to this file?

Reply:
There were at one point in time, but not any more. I can try to fix the git dif here.

Reply:
If it's helpful, you can also do `git checkout upstream/main <filename>` from the command line to get back to the pre-PR version.  (I use it a lot 🙂)